libraries/core/src/test/java/com/paritytrading/nassau/soupbintcp/SoupBinTCPClientStatus.java

```diff
@@ -21,8 +22,8 @@ public SoupBinTCPClientStatus() {
 
     @Override
     public void loginAccepted(SoupBinTCPClient client, SoupBinTCP.LoginAccepted payload) {
-        String session        = ASCII.get(payload.session);
-        long   sequenceNumber = ASCII.getLong(payload.sequenceNumber);
+        String session        = new String(payload.session, US_ASCII);
+        long   sequenceNumber = Long.parseLong(new String(payload.sequenceNumber, US_ASCII).trim());
```

Comment:
FWIW, I like the `ASCII.get` and `ASCII.getLong` methods better, and I suspect `ASCII.getLong` is more efficient, too (though it probably doesn't matter much).